acstech/doppler-events
# Peterk/cb update

Updated the test send file and the client server. The couchbase connection is now a part of the server struct and is made once in the server. Which allows for more data to be sent without fail (fixes #53 ).
To test:
Run `docker-compose up -d` 
Run`cmd/grpcTEST/serviceStart,go`
Run `cmd/testSend/testsend.go`
Wait about a minute and make sure that the data is still being sent.
